builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -425,10 +425,6 @@ node() {
                   error "Deployment targets are not defined"
               }
 
-              if (domain != "jazz") {
```

Comment:
Are we getting rid of subscription filters?

Reply:
No, it's above the code. Actually it is duplicated because of the merge conflicts